community/cypher/cypher/src/main/java/org/neo4j/cypher/internal/tracing/TimingCompilationTracer.java

```diff
@@ -163,24 +163,24 @@ public String query()
 
     private static class Phase extends Event implements PhaseEvent, CompilationPhaseTracer.CompilationPhaseEvent
     {
-        private final CompilationPhaseTracer.CompilationPhase phase;
+        private final CompilationPhaseTracer.CompilationPhase compilationPhase;
```

Comment:
Why?

Reply:
See comment on Query

Reply:
Again, I don't think there was confusion in this case. I don't oppose the change though, just questioning why we are wasting our time with such irrelevant details.